app/Http/Controllers/Select/EventlogController.php

```diff
@@ -68,6 +68,6 @@ protected function baseQuery($request)
             $query->where('device_id', $device_id);
         }
 
-        return $query;
+        return $query->orderBy('type');
```

Comment:
@SourceDoctor shouldn't it be $request->get('field') instead of a hard coded type?  (also you have to protect against sql injection if you do that)

Reply:
tried it, but didn't accept it (crashed because of unknown column)

Reply:
production.ERROR: SQLSTATE[42S22]: Column not found: 1054 **Unknown column 'field' in 'order clause'** (SQL: select distinct `type` from `eventlog` where `device_id` = 2 order by `field` asc limit 51 offset 0) {"userId":2,"exception":"[object] (Illuminate\\Database\\QueryException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'field' in 'order clause' (SQL: select distinct `type` from `eventlog` where `device_id` = 2 order by `field` asc limit 51 offset 0) at /opt/librenms/vendor/laravel/framework/src/Illuminate/Database/Connection.php:678)

Reply:
this is just the string field instead of the value from the request...  check `dd($request->all());`